griffon/commands/entities/osidb/affects/cvss.py

```diff
@@ -0,0 +1,229 @@
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(AffectCVSS))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(AffectCVSS))},
+        "include_meta_attr": {"type": click.Choice(OSIDBService.get_meta_attr_fields(AffectCVSS))},
+    },
+)
+@click.pass_context
+@progress_bar
+def list_affect_cvss(ctx, affect_id, **params):
+    # TODO: handle pagination
+    # TODO: handle output
+    session = OSIDBService.create_session()
+
+    params = multivalue_params_to_csv(params)
+    data = session.affects.cvss_scores.retrieve_list(affect_id, **params).results
+    return cprint(data, ctx=ctx)
+
+
+@affect_cvss.command(name="create")
+@click.option(
+    "--affect-id",
+    "affect_id",
+    help="affect CVE-ID or UUID.",
+    required=True,
+)
+@request_body_options(
+    endpoint_module=osidb_api_v1_affects_cvss_scores_create,
+    exclude=["uuid", "created_dt"],
+)
+@click.pass_context
+def create_affect_cvss(ctx, affect_id, **params):
+    request_body_type = getattr(osidb_api_v1_affects_cvss_scores_create, "REQUEST_BODY_TYPE", None)
+    if request_body_type is None:
+        raise click.ClickException(
+            "No request body template for affect CVSS create. "
+            "Is correct version of osidb-bindings installed?"
+        )
+
+    fields = filter_request_fields(
+        request_body_type.get_fields(),
+        exclude=["uuid", "created_dt"],
+    )
+    params = multivalue_params_to_csv(params)
+
+    session = OSIDBService.create_session()
+
+    data = {field: "" for field in fields}
+    data.update((field, value) for field, value in params.items() if value is not None)
+
+    if ctx.obj["EDITOR"]:
+        data = click.edit(
+            text=json.dumps(data, indent=4, default=str), editor=get_editor(), require_save=False
+        )
+        data = json.loads(data)
+
+    try:
+        data = session.affects.cvss_scores.create(data, affect_id=affect_id)
+    except HTTPError as e:
+        if ctx.obj["VERBOSE"]:
+            console.log(e, e.response.json())
+        raise click.ClickException(
+            "Failed to create affect CVSS. "
+            "You might have insufficient permission or you've supplied malformed data. "
+            "Consider running griffon with -v option for verbose error log."
+        )
+    return cprint(data, ctx=ctx)
+
+
+@affect_cvss.command(name="update")
+@click.option(
+    "--affect-id",
+    "affect_id",
+    help="affect CVE-ID or UUID.",
+    required=True,
+)
+@click.option("--uuid", "cvss_uuid", help="CVSS UUID.", required=True)
+@request_body_options(endpoint_module=osidb_api_v1_affects_cvss_scores_update, exclude=["uuid"])
+@click.pass_context
+def update_affect_cvss(ctx, affect_id, cvss_uuid, **params):
+    request_body_type = getattr(osidb_api_v1_affects_cvss_scores_update, "REQUEST_BODY_TYPE", None)
+    if request_body_type is None:
+        raise click.ClickException(
```

Comment:
makes me wonder if we ever want to wrap up ClickException in a GriffonException ... just a random muttering

Reply:
Might be a good idea. Can be done as a side effect of some future PR. Will make a note. Thanks!